src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -61,7 +61,7 @@ public class Arm extends SubsystemBase {
     public static final double MIN_INCHES = 35.112d;
     public static final double MAX_INCHES = 50.0d;
 
-    public static final double ARM_THETA_DELTA_MODIFIER = 1.0d * ((2 * Math.PI) / 360.0d); // radians
```

Comment:
Let's leave the `1.0d` to be clear that our delta is one degree converted to radians

Reply:
Done